R/comparePlot.R

```diff
@@ -51,23 +52,16 @@
 #' @param symbolFill Set the fill color for the symbols. Note only symbols 21-25 are fillable. This will
 #'        have no effect on other symbols. Default = c("darkgoldenrod1", "deepskyblue4", "red3", "grey25")
 #' @param alpha Controls the transparency of the plotted points (0-1; Default = 0.5)
-#' @param rugColor Specify color for rug density plot along X and Y axes. Set to NULL
-#'        to disable rug layer. (Default = NULL)
-#' @param rugAlpha Sets the transparency for the rug layer. alpha <1.0 takes
-#'        a long time to draw, so for a final plot try 0.1 or 0.2 for alpha for more informative rug.
 #' @param crosshair Color for the crosshair; (Default = "grey50", NULL disables)
 #'        See \url{http://research.stowers-institute.org/efg/R/Color/Chart}
 #' @param referenceLine Color for a slope=1, intercept=0 reference line
 #'        (Default = "darkgoldenrod1"; NULL disables)
 #' @param refLineThickness Set thickness for crosshair and referenceLine (Default = 1)
-#' @param dens2D Overlay a density2D layer (Default = TRUE) Note: only applies
-#'        to simple compare plot without p-value coloring.
 #' @param legendPosition One of "top", "bottom", "left", "right", "ne", "se", "nw", "sw", NULL.
 #'        top/bottom/left/right place the legend outside the figure.  ne/se/nw/sw place the figure
 #'        inside the figure. NULL disables the legend. Default = "right"
-#' @param baseFontSize The smallest size font in the figure in points. Default = 12
-#' @param themeStyle "bw" or "grey" which corresponds to theme_bw or theme_grey respectively.
-#'        Default = "bw"
+#' @param theme For plotType = "ggplot" available options are "bw" or "grey" which corresponds to theme_bw or theme_grey respectively. whereas for plotType = "canvasxpress" use Cx themes.
```

Comment:
I'm not sure why this is bw/gray for the only options, why is this limited?

Reply:
They are from original functions. And I did not find any specific reason why it's limited to only two options.

Reply:
Added task for this https://3.basecamp.com/3775257/buckets/16965597/todos/3409235357